ims/company/acp.md

```diff
@@ -0,0 +1,22 @@
+# Access Control Policy
+
+The Company Access Control Register (ACR) is the authoritative list of who can access what at a company level. It is maintained by the IMS Manager.
```

Comment:
Prefer full name over acronyms, "IMS"

Reply:
IMS Manager is the role title, and is explained in the IMS `README.md`

Reply:
Let's link to it, so that people understand. Or we remove it. IMS Manager doesn't mean much to someone outside of Made Tech.